packages/bindings/src/msg.rs

```diff
@@ -3,23 +3,12 @@ use serde::{Deserialize, Serialize};
 
 use crate::types::SwapAmountWithLimit;
 use crate::{Step, Swap};
-use cosmwasm_std::{CosmosMsg, CustomMsg, Uint128};
+use cosmwasm_std::{CosmosMsg, CustomMsg};
 
 #[derive(Serialize, Deserialize, Clone, PartialEq, JsonSchema, Debug)]
 #[serde(rename_all = "snake_case")]
 /// A number of Custom messages that can call into the Osmosis bindings
 pub enum OsmosisMsg {
-    /// Contracts can mint native tokens that have an auto-generated denom
-    /// namespaced under the contract's address. A contract may create any number
-    /// of independent sub-denoms.
-    /// Returns FullDenomResponse in the data field of the Response
-    MintTokens {
```

Comment:
OK, but wouldn't make more sense to keep this message here, and disable it on the Go side? Perhaps just put a comment here.

Anyway I think I follow your logic here: let's add stuff when it's working. And, we can always rely on the git commit history.